src/executor/cursor.rs

```diff
@@ -1116,14 +1939,15 @@ mod tests {
                 r#"OR ("t"."col_1" = 'val_1' AND "t"."col_2" = 'val_2' AND "t"."col_3" > 'val_3')"#,
                 r#"OR ("t"."col_1" = 'val_1' AND "t"."col_2" > 'val_2')"#,
                 r#"OR "t"."col_1" > 'val_1'"#,
+                r#"ORDER BY "t"."col_1" DESC, "t"."col_2" DESC, "t"."col_3" DESC, "t"."col_4" DESC, "t"."col_5" DESC"#,
             ].join(" "))
         );
 
         assert_eq!(
             DbBackend::Postgres.build(&
                 Entity::find()
                 .cursor_by((Column::Col1, Column::Col2, Column::Col3, Column::Col4, Column::Col5, Column::Col6))
-                .after(("val_1", "val_2", "val_3", "val_4", "val_5", "val_6"))
+                .before(("val_1", "val_2", "val_3", "val_4", "val_5", "val_6")).desc().apply_limit().apply_order_by().apply_filters()
```

Comment:
Same as my previous comment, it's a copy of `cursor_by_many` named as `cursor_by_many_desc`, no tests were removed or changed except additions like `.apply_limit().apply_order_by().apply_filters()` because filters applied when we call `all()`